lib/proj-parser.js

```diff
@@ -0,0 +1,77 @@
+var fs = require('fs');
+var path = require('path');
+var parseXML = require('xml2js').parseString;
+var debug = require('debug')('snyk');
+
+function determineDotnetVersion(rootDir) {
+  var csprojPath = findFile(rootDir, /.*\.csproj$/);
+  if (!csprojPath) {
```

Comment:
is there is any test for this scenario?

Reply:
yes, it is the `'parse dotnet with vbproj'`